src/main/java/org/folio/marc4ld/service/marc2ld/Marc2ldFieldRuleImpl.java

```diff
@@ -0,0 +1,140 @@
+package org.folio.marc4ld.service.marc2ld;
+
+import static java.lang.String.join;
+import static org.apache.commons.lang3.StringUtils.SPACE;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NonNull;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.configuration.property.Marc4BibframeRules;
+import org.folio.marc4ld.service.marc2ld.field.property.PropertyRule;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+
+@Builder
+@AllArgsConstructor
+public class Marc2ldFieldRuleImpl implements Marc2ldFieldRule {
+
+  private final String label;
+  private final Relation relation;
+  @NonNull
```

Comment:
What's the point of this annotation on the field level? It's said:
"If put on a field, any generated method assigning a value to this field will also produce these null-checks."
and there is no generated methods, right?

Reply:
Discussed
it's lombok annotation for lombok `Builder`